### PR TITLE
quicker dev build

### PR DESCRIPTION
**Pull request by janpio**
use exclude/include to only build 3 languages in dev 
instead of using exclusion list that always has to be updated

### CODE REVIEW

File: conf/_dev.yml [hunk 1]
```diff
@@ -1,107 +1,13 @@
...
-    - docs/zh-tw/3.5.0
-    - docs/zh-tw/5.4.0
-    - docs/zh-tw/6.x
-    - docs/zh-tw/dev
+    - docs/
+
+include:
+    - docs/en/dev
+    - docs/ru/dev
+    - docs/fr/dev
```
Comment (dblotsky) [inline conf/_dev.yml]: Nitpick: no newline at the end of the file.

File: conf/_dev.yml [hunk 2]
```diff
@@ -1,107 +1,13 @@
...
-    - docs/zh-cn/dev
-    - docs/zh-tw/3.1.0
-    - docs/zh-tw/3.4.0
-    - docs/zh-tw/3.5.0
-    - docs/zh-tw/5.4.0
-    - docs/zh-tw/6.x
-    - docs/zh-tw/dev
+    - docs/
+
+include:
```
Comment (dblotsky) [inline conf/_dev.yml]: It might also be worth it to add the `7.x` or `6.x` versions, so that it's easier to test some links.

Comment (janpio) [reply conf/_dev.yml]: Would having those for `en` be enough?

Comment (dblotsky) [reply conf/_dev.yml]: @janpio so, I'd prefer these:
```
en/dev
ru/dev
fr/dev
en/7.x
ru/7.x
fr/7.x
```

**Comment by Menardi** [comment]
I was just about to create a PR for this. I wish this had already been merged, it would have saved me a lot of time! Is there are reason it hasn't been merged yet?

**Comment by janpio** [comment]
See the second comment. Also I am not sure if the release of the 8.x docs has changed anything that would have to be adapted here. Feel free to drive this forward (as I am not free right now)!

**Comment by dblotsky** [comment]
@janpio this looks good to merge, even without the other versions.

**Pull request merged**